### PR TITLE
docs: update wording of nodejs prerequisite

### PR DESCRIPTION
**Pull request by blackchoey**

### CODE REVIEW

File: NPM-search-connector-M365/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ NPM Search Connector is a Messaging Extension that allows you to perform a quick
 ![Npm Search Connector](images/npm-search-connector-M365.gif)
 
 ## Prerequisite
-- [NodeJS](https://nodejs.org/en/), version 10, 12 or 14
+- [NodeJS](https://nodejs.org/en/), fully tested on NodeJS 10, 12, 14
```
Comment (kuojianlu) [inline NPM-search-connector-M365/README.md:14]: `fully tested on NodeJS 14` to align with other samples.

File: todo-list-with-Azure-backend-M365/README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Todo List app helps to manage your personal to do items. This app can be install
 ![Todo Item List in Office](images/todo-list-M365-in-office.gif)
 
 ## Prerequisite
-- [NodeJS](https://nodejs.org/en/), version 10, 12 or 14
+- [NodeJS](https://nodejs.org/en/), fully tested on NodeJS 10, 12, 14
```
Comment (kuojianlu) [inline todo-list-with-Azure-backend-M365/README.md:24]: `fully tested on NodeJS 14` to align with other samples.

**Comment by kuojianlu** [comment]
Please note that the Nodejs version dependency of `todo-list-with-Azure-backend-M365` is the same as that of `todo-list-with-Azure-backend`, while the Nodejs version dependency of `NPM-search-connector-M365` is the same as that of `hello-world-bot`.

**Comment by blackchoey** [comment]
> Please note that the Nodejs version dependency of `todo-list-with-Azure-backend-M365` is the same as that of `todo-list-with-Azure-backend`, while the Nodejs version dependency of `NPM-search-connector-M365` is the same as that of `hello-world-bot`.

Thanks, since you have tested these 2 samples on nodejs 12, I will note they're fully tested on this version. For other samples, we will test the samples on nodejs 12 in next sprint and update the prerequisites.

**Review by zyxiaoyuer — APPROVED** [review]

**Pull request merged**